daemon/daemon.go

```diff
@@ -593,6 +593,11 @@ func New() (*Daemon, error) {
 	if err != nil {
 		return nil, err
 	}
+	err = ovld.EnableHotplug()
+	if err != nil {
+		logger.Noticef("Failed to enable hotplug: %s", err)
```

Comment:
I'm curious why we are logging it? The error will be handled by something, right?